index 09f25a3..c5caab3 100644
--- a/data/org.gnome.FileRoller.desktop.in.in
+++ b/data/org.gnome.FileRoller.desktop.in.in
@@ -9,9 +9,8 @@ StartupNotify=true
 Terminal=false
 Type=Application
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
-Icon=org.gnome.ArchiveManager
+Icon=utilities-file-archiver
 Categories=GTK;GNOME;Utility;Archiving;Compression;X-GNOME-Utilities;
-NotShowIn=KDE;
 MimeType=@MIMETYPES@
 X-GNOME-DocPath=file-roller/file-roller.xml
 X-GNOME-UsesNotifications=true
