index 09f25a3..04d79da 100644
--- a/data/org.gnome.FileRoller.desktop.in.in
+++ b/data/org.gnome.FileRoller.desktop.in.in
@@ -11,7 +11,6 @@ Type=Application
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=org.gnome.ArchiveManager
 Categories=GTK;GNOME;Utility;Archiving;Compression;X-GNOME-Utilities;
-NotShowIn=KDE;
 MimeType=@MIMETYPES@
 X-GNOME-DocPath=file-roller/file-roller.xml
 X-GNOME-UsesNotifications=true
